### ISSUE TITLE
v3.5.1 breaks build process with gulp

### ISSUE THREAD

**Issue by cgrafton** [ben-eb/cssnano#133]
With the newest version of cssnano, we see the following error when using gulp to build the package. Here is the stack trace:

```build	10-Feb-2016 19:31:32	events.js:85
build	10-Feb-2016 19:31:32	      throw er; // Unhandled 'error' event
build	10-Feb-2016 19:31:32	            ^
build	10-Feb-2016 19:31:32	[0m[91mTypeError: Cannot read property 'value' of undefined
build	10-Feb-2016 19:31:32	    at /tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/dist/lib/reducePositions.js:72:49
build	10-Feb-2016 19:31:32	    at Array.forEach (native)
build	10-Feb-2016 19:31:32	    at transform (/tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/dist/lib/reducePositions.js:64:14)
build	10-Feb-2016 19:31:32	    at /tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/node_modules/postcss/lib/container.js:88:34
build	10-Feb-2016 19:31:32	    at /tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/node_modules/postcss/lib/container.js:73:26
build	10-Feb-2016 19:31:32	    at Rule.each (/tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/node_modules/postcss/lib/container.js:60:22)
build	10-Feb-2016 19:31:32	    at Rule.walk (/tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/node_modules/postcss/lib/container.js:72:21)
build	10-Feb-2016 19:31:32	    at /tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/node_modules/postcss/lib/container.js:76:32
build	10-Feb-2016 19:31:32	    at Root.each (/tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/node_modules/postcss/lib/container.js:60:22)
build	10-Feb-2016 19:31:32	    at Root.walk (/tmp/fulfill/node_modules/gulp-cssnano/node_modules/cssnano/node_modules/postcss/lib/container.js:72:21)```

**Comment by ben-eb** [comment]
Can you attach some CSS that can reproduce the problem, please? As a quick solution, you can disable the module `{reducePositions: false}`

**Comment by cgrafton** [comment]
I didn't see that you released 3.5.1, I meant to mention 3.5.0, released yesterday afternoon. I just tried it with 3.5.1 (haven't build since you released that) and it looks like the issue is fixed with the most recent version. 

Thank you.

**Comment by ben-eb** [comment]
Great! :+1:
